src/org/parosproxy/paros/view/FindDialog.java

```diff
@@ -87,16 +99,24 @@ public FindDialog(Frame arg0, boolean arg1) throws HeadlessException {
      * @param parent the parent window of the FindDialog.
      * @param modal whether or not this FindDialog should be modal
      * @throws HeadlessException
+     * @since TODO add version
      */
     public FindDialog(Window parent, boolean modal) throws HeadlessException {
     	super(parent, modal);
-    	initialize();
+
+    	initialize(parent.toString());

```

Comment:
`toString()` is not reliable, for example, if the parent window is resized it will be a different "name" (thus opening another find dialogue).